app/workers/follows/create_chat_channel_worker.rb

```diff
@@ -1,6 +1,7 @@
 module Follows
-  class CreateChatChannelJob < ApplicationJob
-    queue_as :create_chat_channel_after_follow
+  class CreateChatChannelWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :default, retry: 10
```

Comment:
Can we put this in the low_priority queue? I know that the default has the same priority but I find that people often get confused when they see the default queue, whereas the low_priority queue is more explicit about what level of priority it is.

Reply:
it's a fair point, I agree. I will move it